### ISSUE TITLE
Archive links for auction links

### ISSUE THREAD

**Issue by Poliwrath** [Shizmob/arcade-docs#16]
YAJ listing pages don't last very long (unsure how long eBay ones last), and aucfan/the billion other proxy sites that save YAJ pages aren't guaranteed to keep them.
This could be extended for *everything*, but most sources have been on the internet (i.e iMp95's site) for ages, so the likelihood of them suddenly disappearing is probably low...

**Comment by Shizmob** [comment]
Yep, agree we need some work here. I'm also a fan of using `https://aucview.aucfan.com/yahoo/<auction ID>` to be able to check deleted auctions/removed images, but even that has its limitations.

**Comment by Poliwrath** [comment]
Some YAJ sellers go the extra step to delete auction images after the listing is over (before the 6 month auto auction delete or whatever) fwiw. I don't exactly have a known list of sellers that do that on hand but I do know ``cyberdaioo`` does it sometimes.

**Comment by voidderef** [comment]
We already talked about a even more general solution of automatically crawling links regularly and persisting the files, e.g. using a github action cronjob that runs once a day. 

One option to get this idea started might be to focus on the yahoo auction links first, and explore it with the limited scope. There might be a bunch of useful learnings from that before this can/should be scaled further.

**Comment by cglmrfreeman** [comment]
Any updates to this? I am already finding [dead links](https://github.com/Shizmob/arcade-docs/blame/main/sega/boards.md#L8) in the Sega boards section.
